qemu/tests/usb_redir.py

```diff
@@ -72,6 +84,35 @@ def _host_config_check():
                 err_msg = "Fail to install 'virt-viewer' on host, "
                 err_msg += "output: %s" % o
                 return (status, err_msg)
+        elif backend == 'tcp_socket':
+            usbredir_cmd = "rpm -q usbredir || yum install -y usbredir"
+            s, o = process.getstatusoutput(usbredir_cmd, shell=True)
+            if s:
+                status = False
+                err_msg = "Fail to install 'usbredir' on host, "
+                err_msg += "output: %s" % o
+                return (status, err_msg)
+            usbredirserver_query = "rpm -q usbredir-server"
+            s, o = process.getstatusoutput(usbredirserver_query, shell=True)
+            if s:
+                usbredir_query = "rpm -q usbredir"
+                usbredir_ver = process.getoutput(usbredir_query, shell=True)
+                (build, arch) = usbredir_ver.rsplit(".", 1)
+                brew_cmd = "brew download-build --arch=%s %s" % (arch, build)
```

Comment:
`brew` is internal, it is better do not use it. Have you tried to find a repo that include `usbredir-server` pacakge ?

Reply:
Updated, thanks!